workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/CodeGenerator.cs

```diff
@@ -58,17 +58,17 @@ public int Run()
             }
 
             GenerateNativeTypesAndAst();
-
+            return 0;
```

Comment:
what about the lines underneath this. they will never get executed, right?

Reply:
Was intentional for debugging - but yeah should be removed here 😂